client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -149,10 +153,22 @@ export class DisplayTable extends React.Component {
       .map(_.last)
       .value();
 
+    const data_csv = [ordered_column_keys].concat(
```

Comment:
I'm still questioning about this line. If `ordered_column_keys` is keys, then aren't you using internal/uni-lingual keys as the headers of the downloaded/copied csv? I only checked one or two and they looked like they used the correct plain language headers, which is why I asked if we were using the plain language column names as keys.

Reply:
Oh shoot, I misinterpreted your question earlier. You're totally right, I'm putting keys into CSV as headers, I actually didn't even notice that, I guess I got way too familiar with these DisplayTable keys

Reply:
Ah! Ok, I was asking it the other way around. I need to double check the case I tested because I swear the headers WERE plain language and not keys, haha.

Reply:
Oh, damn, right. I was looking at graph tables, and for dumb nivo reasons we DO use plain language names as keys in those. The other tables would have been more obvious, yeah.

Reply:
Oh haha, maybe Kate changed them around? I THINK they were keys ever since I started working on DisplayTable (back when I implemented search filter). Anyways, I pushed changes to fix this one